src/map.cpp

```diff
@@ -5187,6 +5189,52 @@ const trap &map::tr_at( const tripoint &p ) const
     return current_submap->get_trap( l ).obj();
 }
 
+bool map::crafting_bill_nearby( const tripoint &p, int range )
+{
+    for( const tripoint &elem : g->m.points_in_radius( p, range ) ) {
```

Comment:
Why `g->m`? What happens when this function is called on a separate `map` object?
```suggestion
    for( const tripoint &elem : points_in_radius( p, range ) ) {
```